tools/cmake/preset/default.cmake

```diff
@@ -227,13 +237,41 @@ define_overridable_option(
 set(_default_executorch_build_executor_runner ON)
 if(APPLE AND "${SDK_NAME}" STREQUAL "iphoneos")
   set(_default_executorch_build_executor_runner OFF)
+elseif(DEFINED EXECUTORCH_BUILD_PRESET_FILE)
+  set(_default_executorch_build_executor_runner OFF)
 endif()
 define_overridable_option(
   EXECUTORCH_BUILD_EXECUTOR_RUNNER
   "Build the executor_runner executable"
   BOOL ${_default_executorch_build_executor_runner}
 )
 
+# NB: Enabling this will serialize execution of delegate instances Keeping this
+# OFF by default to maintain existing behavior, to be revisited.
+define_overridable_option(
+  EXECUTORCH_XNNPACK_SHARED_WORKSPACE
+  "Enable workspace sharing across different delegate instances"
+  BOOL ON
```

Comment:
Shouldn't this be OFF by default, based on the comment?

Reply:
It's currently on by default:

https://github.com/pytorch/executorch/blob/a0d9c7e68fa9c36c62b6849fd606622ff72a7c0a/backends/xnnpack/CMakeLists.txt#L28-L32

I'm just moving the option definition here from `backends/xnnpack/CMakeLists.txt`